src/AudacityApp.cpp

```diff
@@ -373,6 +377,35 @@ void PopulatePreferences()
    gPrefs->Flush();
 }
 
+#if defined(USE_BREAKPAD)
+void InitBreakpad()
```

Comment:
I think need move this function into some `breakpad` folder, because I don't see a reason to declarate it in `AudacityApp.cpp`, because `InitBreakpad` not dependenly (not using) of various `AudacityApp` constructions.

Reply:
It will bring lots of dependencies into crashreports module, and  InitBreakpad's job is to configure environment (which is defined by application) for breakpad to run.

Reply:
I don't understand @gera-gas.  InitBreakpad() is in fact defined in another folder.  AudacityApp must call it.

@vsverchinsky makes the point (as I also noted in comments about the CMakeLists.txt files) that the breakpad folder makes a static library for Audacity, and that library itself depends on other libraries.  CMake variables make it easy to eilminate all of that from a build if that is desired.